test/RPCSession.h

```diff
@@ -126,6 +126,7 @@ class RPCSession: public boost::noncopyable
 
 	IPCSocket m_ipcSocket;
 	size_t m_rpcSequence = 1;
+	unsigned m_maxMiningTime = 15000; // 15 seconds
```

Comment:
What is the best timeout to choose here? 60 seconds? The old code could have gone up ridiculous amounts by failing every poll and doubling the time.